.editorconfig

```diff
@@ -0,0 +1,26 @@
+# For more information about the properties used in
```

Comment:
Any reason for including this file? I don't mind, but I usually only have it in a project root, not a module?